### PR TITLE
Remove fixed size buffer when reading lines from the response file.

### PR DESCRIPTION
**Pull request by jlaanstra**
This PR removes a hardcoded maximum line length and instead, dynamically allocates a string to read lines. 

Currently cppwinrt.exe silently fails. @kennykerr thoughts on outputting some kind of error when the response file contains invalid data?

### CODE REVIEW

**Comment by kennykerr** [comment]
You can report errors using `throw_invalid()`.

**Comment by jlaanstra** [comment]
Looks like that might be handled already as it hits the same code path as if it was passed in without a response file.

File: src/library/cmd_reader.h [hunk 1]
```diff
@@ -269,9 +269,9 @@ namespace xlang::cmd
             {
                 throw_invalid("'@' is reserved for response files");
             }
-            std::array<char, 8192> line_buf;
+            std::string line_buf;
             std::ifstream response_file(absolute(response_path));
-            while (response_file.getline(line_buf.data(), line_buf.size()))
+            while (getline(response_file, line_buf)) // getline calls line_buf.erase()
```
Comment (Scottj1s) [inline src/library/cmd_reader.h]: comment unnecessary

Comment (jlaanstra) [reply src/library/cmd_reader.h]:
```suggestion
            while (getline(response_file, line_buf))
```

**Comment by kennykerr** [comment]
Thanks Johan!

**Pull request merged**